exercises/practice/hello-world/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if [ ! -f ../../../bin/cobolcheck ]; then
+    echo "Cobolcheck not found, try to fetch it."
+    DIR=$(pwd)
+    cd ../../../bin/
+    ./fetch-cobolcheck
+    cd $DIR
+fi
+../../../bin/cobolcheck -c ../../../etc/config.properties -p hello-world
```

Comment:
Unfortunately, this relative structure won't work as students only receive the files with the exercise's directory. This means that any files the student needs have to be duplicated within the exercise directory.

A nice way to verify this is by joing the cobol track (which I've just added to the database) by going to https://exercism.org/tracks/cobol (the track is only visible to maintainers and not listed in the tracks list). Once joined, you can use the CLI to download the exercise locally (e.g. `exercism download --exercise=hello-world --track=cobol`) to inspect if things would work for students.